### PR TITLE
Update to tokio-1.0

### LINKED ISSUE #2621: Port to tokio 1.0
Need to keep up with the async stack.

## Dependencies needing upgrade

- [x] tonic https://github.com/hyperium/tonic/pull/530 
- [x] ~warp https://github.com/seanmonstar/warp/issues/725~ left on 0.2 because of juniper
- [x] juniper, ~replace with asyng-graphql https://github.com/input-output-hk/jormungandr/issues/2945~ left for the time being with a tokio-compat-02 shim

### PR DESCRIPTION
**Pull request by danielSanchezQ**
May close #2621

### CODE REVIEW

File: jormungandr/Cargo.toml [hunk 1]
```diff
@@ -55,12 +55,13 @@ slog-term = "2.6.0"
...
-tokio-stream = "0.1.0"
+tokio-stream = "0.1.2"
 tokio-util = { version = "0.6.0", features = ["time"] }
-tonic = "0.3"
+tonic = "0.4"
 async-trait = "0.1"
 lru = "^0.6.1"
-warp = { version = "0.2.4", features = ["tls"] }
+warp = { version = "0.3", features = ["tls"] }
+tower = "0.4.4"
```
Comment (Zeegomo) [inline jormungandr/Cargo.toml:64]: Is the explicity `tower` dependency intended?

Comment (danielSanchezQ) [reply jormungandr/Cargo.toml:64]: No, it is a leftover indeed.

**Comment by mzabaluev** [comment]
Please update to latest chain-libs to bring in the latest tonic release, and rebase.

**Comment by danielSanchezQ** [comment]
We need for juniper to be updated. Should be the last missing bit for update all our async stack.

**Comment by mzabaluev** [comment]
Works all right. There is a regression with nightly on unrelated code.

**Comment by danielSanchezQ** [comment]
We can use this for the time being and once everything is updated we can do so with  juniper and syn.

**Pull request merged**